packages/astro-uxds/_content/components/application-state.md

```diff
@@ -35,15 +35,15 @@ The Application State component indicates the state the entire application is in
 
 - The background rectangle does not change height, but will change width with longer text content.
 
-- The default color for the background rectangle uses the `--colorTertiaryDarken4` variable, but can be changed to any of the Tag-approved colors in the Astro palette when more user attention to the component is required.
+- It is suggested that the background color for the application state component uses Astro 7.0 palettes Purple, Hot Orange, Pink, and Teal (commonly used for Tags).
 
 ::: caution
 Only use Tag colors when necessary as overuse of colors may reduce efficacy of monitoring icon, classification marking, and notification banner component colors.
 :::
 
 ### Text
 
-- Text within the component uses the Medium body text size (14 px/0.875 rem) and sentence casing.
+- Text within the component uses the Body 2 style and sentence casing.
 
 - The default font color is the Dark Theme’s default body text color (#FFFFFF) for use with the default, dark background color.
 
```

Comment:
Pretty sure the `--colorTertiaryDarken4` will be dropped and instead be `color.palette.darkblue.950`. Third bullet point in the text section of https://rocketcom.atlassian.net/wiki/spaces/AS11/pages/2612789259/Application+State.

Reply:
Yes! Just updated that, as well as all the other text on the page. Cole said anything (even not in green) needs to be changed. It should all be identical now